package.json

```diff
@@ -116,5 +116,9 @@
     "snapshot:production": "FOUNDRY_PROFILE=production forge snapshot --snap .gas-snapshot-production",
     "lint:diff": "npx eslint --cache --cache-location ./.eslintcache -c .eslintrc.js $(git diff --relative --name-only --diff-filter=d $(git merge-base HEAD origin/master) -- \"*.ts\" \"*.js\" \"*.env\" \"*.toml\")",
     "artifactregistry-login": "npx google-artifactregistry-auth"
+  },
+  "dependencies": {
```

Comment:
should these be devDependencies?

Reply:
mm yeah good catch.  I think I can remove `fireblocks-sdk` and `fireblocks-defi-sdk` while I'm at it too